clip.py

```diff
@@ -99,6 +107,24 @@ def __init__(self, width, height):
         self.is_record = False
         self.outputsPorts = set()
         self.outputsPorts.add(Clip.DEFAULT_OUTPUT)
+        self.scenes = {}
```

Comment:
Since I still find it good to have an association with the names, ill use pythons OrderedDict from collections to guarantee the order.